iceoryx_utils/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
-# Copyright (c) 2019, 2020 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
+# Copyright (c) 2019 - 2021 by Robert Bosch GmbH. All rights reserved.
```

Comment:
@marthtz would you check if the dates are correct here? I set them based on the git file history.
Same for the files below.

Reply:
Yep, all good!